core/templates/ide/clion/CMakeLists.txt.tpl

```diff
@@ -16,8 +16,14 @@ set(CMAKE_CXX_STANDARD {{cmake-cxx-standard}})
 {{add_definitions({{user-definition}})}}
 % end
 
+# add search paths for cosa core
```

Comment:
There are a bunch of things here that are environment specific, like `WCOSA_CMD`, the `include_directories`. These should be put into a `CMakeListsPrivate.txt` which is gitignored (while `CMakeLists.txt` is not) and is generated for each environment. I.e. each person working on a `wcosa` project clones the repo, runs `wcosa init` or something, which grabs the wcosa paths and generates `CMakeListsPrivate.txt`. I'd also just throw all the flags and stuff into there as well just to hide them from the user.

Reply:
Also you need to add `include_directories("${COSA_SDK_PATH}/variants/arduino/${BOARD_NAME}")`, because from what I understand, these are only to help CLion index the files and help with code completion, not for actually building the project (which is done purely through `wcosa/CMakeLists.txt`

Reply:
yeah we won't build through CLion CMakeLists, this is just a dummy for code completion and working on the project